packages/core/platform/templates/helmreleases.yaml

```
@@ -11,8 +11,17 @@
 {{- end }}
 
 {{- range $x := $bundle.releases }}
-{{- if not (has $x.name $disabledComponents) }}
-{{- if or (not $x.optional) (and ($x.optional) (has $x.name $enabledComponents)) }}
+
+{{- $shouldInstall := true }}
+{{- $shouldDelete := false }}
+{{- if or (has $x.name $disabledComponents) (and ($x.optional) (not (has $x.name $enabledComponents))) }}
+  {{- $shouldInstall = false }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)